scripts/alert_emails/template.html

```diff
@@ -166,6 +166,33 @@
                                 </div>
                                 <div
                                     style="line-height:inherit;border-collapse:collapsed;width:100%;border-width:0!important;border-style:none!important;border-color:white!important">
+
+                                    <!-- Disclaimer -->
```

Comment:
I copy / pasted all this html from the "About" section.  The only thing I changed is the template parts (`{{disclaimer.name}}`, `{{disclaimer.text}}`)